javageci-tools/src/main/java/javax0/geci/tools/CompoundParams.java

```diff
@@ -140,6 +140,9 @@ private void checkAllowedKeys() {
      * @return the value of the parameter
      */
     public String get(String key, String defaults) {
+        if (allowedKeys != null && !allowedKeys.contains(key)) {
```

Comment:
It is great catch.

No code it without copy paste.

Reply:
Moved it to get0, because all three methods that *should* throw this, call that.